web-ui/src/components/feedback_template_selector/FeedbackTemplateSelector.jsx

```diff
@@ -85,18 +120,22 @@ const FeedbackTemplateSelector = (props) => {
       />
       }
       <div className="card-container">
-        {getTemplates().map((template) => (
-          <TemplateCard
-            key={template.id}
-            title={template.title}
-            creator={template.creator}
-            description={template.description}
-            isAdHoc={template.isAdHoc}
-            questions={template.questions}
-            expanded={preview.open}
-            onClick={(e) => handlePreviewOpen(e, template)}
-            onCardClick={() => onCardClick(template)}/>
-        ))}
+        {
+          (filteredTemplates.length === 0 && searchText === "")
```

Comment:
JS has truthy and falsy values.  Zeros and empty strings are both falsy meaning they will evaluate to false by default so you can just write it as (!filteredTemplates.length && !searchText).  I wouldn't worry about changing it here but figured I'd mention it as it's a nice time saver.
https://www.sitepoint.com/javascript-truthy-falsy/